ci-operator/config/opendatahub-io/notebooks/opendatahub-io-notebooks-2024a.yaml

```diff
@@ -0,0 +1,471 @@
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: rhel-8-release-golang-1.19-openshift-4.14
+images:
+- context_dir: base/ubi8-python-3.8
+  dockerfile_path: Dockerfile
+  to: base-ubi8-python-3.8
+- context_dir: jupyter/minimal/ubi8-python-3.8
+  dockerfile_path: Dockerfile
+  from: base-ubi8-python-3.8
+  to: jupyter-minimal-ubi8-python-3.8
+- context_dir: jupyter/datascience/ubi8-python-3.8
+  dockerfile_path: Dockerfile
+  from: jupyter-minimal-ubi8-python-3.8
+  to: jupyter-datascience-ubi8-python-3.8
+- context_dir: cuda/ubi8-python-3.8
+  dockerfile_path: Dockerfile
+  from: base-ubi8-python-3.8
+  to: cuda-ubi8-python-3.8
+- context_dir: jupyter/minimal/ubi8-python-3.8
+  dockerfile_path: Dockerfile
+  from: cuda-ubi8-python-3.8
+  to: cuda-jupyter-minimal-ubi8-python-3.8
+- context_dir: jupyter/datascience/ubi8-python-3.8
+  dockerfile_path: Dockerfile
+  from: cuda-jupyter-minimal-ubi8-python-3.8
+  to: cuda-jupyter-datascience-ubi8-python-3.8
+- context_dir: jupyter/trustyai/ubi8-python-3.8
+  dockerfile_path: Dockerfile
+  from: jupyter-datascience-ubi8-python-3.8
+  to: jupyter-trustyai-ubi8-python-3.8
+- context_dir: habana/1.10.0/ubi8-python-3.8
+  dockerfile_path: Dockerfile
+  from: jupyter-datascience-ubi8-python-3.8
+  to: habana-jupyter-1.10.0-ubi8-python-3.8
+- context_dir: habana/1.11.0/ubi8-python-3.8
+  dockerfile_path: Dockerfile
+  from: jupyter-datascience-ubi8-python-3.8
+  to: habana-jupyter-1.11.0-ubi8-python-3.8
```

Comment:
lets remove habana 1.11.0 build and test 
as we no longer provide it in manifests